ourpackage/veslino.py

```diff
@@ -0,0 +1,72 @@
+import os
+import random
+
+def marc():
+    os.system("cls")
+    while True:
+        print(f"\033[1;35m{'=' * 50}")
+        print("Hello, I'm Marc D. Vesliño")
+        print("1. Basic Information")
+        print("2. Goals")
+        print("3. Guessing Game")
+        print("4. Comment from Barcelos")
+        print("5. Comment from Consultado")
+        print("6. Comment from Andaya")
+        print("0. Exit")
```

Comment:
I would like to add a comment about your program - Piadozo

Reply:
You have done it.